src/pages/policy/input/ParameterEditor.jsx

```diff
@@ -30,6 +31,17 @@ export default function ParameterEditor(props) {
     startDate,
   );
 
+  useEffect(() => {
+    const [parameterStartDate, parameterEndDate] = policy.reform.data[
```

Comment:
Can you refactor this effect hook to only run when `policy.reform.data[parameterName]` exists, then run your setters based upon that? At the moment, this hook re-sets `startDate` and `endDate` each time we access a new, unset parameter